openmdao/core/group.py

```diff
@@ -398,6 +405,8 @@ def _setup_procs(self, pathname, comm, mode):
         # build a list of local subgroups to speed up later loops
         self._subgroups_myproc = [s for s in self._subsystems_myproc if isinstance(s, Group)]
 
+        self._loc_subsys_map = {s.name: s for s in self._subsystems_myproc}
```

Comment:
This variable doesn't seem to be documented either.

Reply:
fixed